### PR TITLE
Add event routes and schema

### PR DESCRIPTION
**Pull request by jbitton**
Let me know what you think!

Tested via Postman

### CODE REVIEW

File: .idea/jsLibraryMappings.xml [hunk 1]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lexlevi17) [inline .idea/jsLibraryMappings.xml]: don't commit these files ples (.xml, .iml)

File: routes/events.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  db.Event.find({ 'groupId': ObjectId(req.params.groupId) }, function (err, events) {
+    if (err || events === null)
+      res.status(400).send([]);
+    else
+      res.status(200).send(events);
+  });
+});
+
+// POST a new event for a Group
+router.post('/new/:groupId', function(req, res) {
```
Comment (lexlevi17) [inline routes/events.js]: if you wanna be consistent about route naming, do /:groupId/new, like you did w/ delete

**Review by alevkov — COMMENTED** [review]
review comments

File: .idea/jsLibraryMappings.xml (hunk 1 above)
Comment (jbitton) [inline .idea/jsLibraryMappings.xml]: okay, I just committed them bc you include the .idea/ lib in your repo

File: routes/events.js (hunk 2 above)
Comment (jbitton) [inline routes/events.js]: okay!

**Review by alevkov — APPROVED** [review]
lgtm

**Pull request merged**